src/model/events-model.js

```diff
@@ -0,0 +1,39 @@
+import { getMockEvents } from '../mock/event.js';
+import { getMockDestinations } from '../mock/destination.js';
+import { getMockOffers } from '../mock/offer.js';
+
+export default class EventsModel {
+  constructor(){}
+  events = getMockEvents();
+  destinations = getMockDestinations();
+  offers = getMockOffers();
+
+  getEvents() {
+    return this.events;
+  }
+
+  getDestinations() {
+    return this.destinations;
+  }
+
+  getOffers() {
+    return this.offers;
+  }
+
+  getDestination = (id) => {
+    if (id) {
+      return this.destinations.find((element) => element.id === id);
+    }
```

Comment:
Избыточная проверка, потому что если не передавать `id`, то `find` всё равно вернёт undefined

```suggestion
    return this.destinations.find((element) => element.id === id);
```